service/settingsservice.php

```diff
@@ -17,8 +17,9 @@ class SettingsService
 	private $root;
 
 	/* Default values */
-	private $settings = [
+	private $defaults = [
 		"notesPath" => "Notes",
+		"fileSuffix" => ".txt",
```

Comment:
please make this value globally changeable. this is the most important part of the issue your are fixing here.
maybe even configuring the `notesPath` globally would be nice.

Reply:
This PR is about a per user setting. I think that we should do a system-wide (admin) setting in a separate PR (if it is really needed).